libraries/AP_SerialManager/AP_SerialManager.h

```diff
@@ -72,6 +72,9 @@
 #define AP_SERIALMANAGER_SBUS1_BUFSIZE_RX     16
 #define AP_SERIALMANAGER_SBUS1_BUFSIZE_TX     32
 
+#define AP_SERIALMANAGER_DEVO_TELEM_BAUD        38400
+#define AP_SERIALMANAGER_DEVO_BUFSIZE_RX        0
+#define AP_SERIALMANAGER_DEVO_BUFSIZE_TX        32
```

Comment:
Please move this to the DEVO_Telem cpp file, that's where this is used.

Reply:
This done as it done to all another needs, eg. Frsky

Reply:
No, it's not the same. The FrSky is used in AP_SerialManager.cpp. Although it's a dumb use of it, I'll let it pass if you want to do the same they did.

Reply:
OK